### ISSUE TITLE
Not compatible with ida 9.0.

### ISSUE THREAD

**Issue by Sexy-Frog** [pgarba/SwitchIDAProLoader#10]
We are waiting.

**Comment by pgarba** [comment]
Need to wait until the official release ;)

**Comment by break-core** [comment]
IDA 9.0 was just released officially around a week ago.

**Comment by pgarba** [comment]
Yep, waiting for the company to get the new version and SDK. Once it's
there I will update it

riall ***@***.***> schrieb am Fr., 11. Okt. 2024, 00:47:

**Comment by goofyclown** [comment]
bump?

**Comment by fm360** [comment]
ida 9.1 is out now. maybe publishing the source code of what you have would help a little?

**Comment by dmiller423** [comment]
yes,  I would have fixed it by now with the code avail...
